ui/src/services/renderTemplate.ts

```diff
@@ -12,20 +20,36 @@ export type VoteData = {
 export function renderTemplate(voteData?: VoteData) {
   if (!voteData)
     return `Oops, something went wrong. We got no data to show you, try reloading and contacting dreamlabs if this keeps happening`;
-  const keys = ["great", "ok", "notThatGreat", "notGreatAtAll"];
+  const keys: (keyof CounterData)[] = [
+    "great",
+    "ok",
+    "notThatGreat",
+    "notGreatAtAll"
+  ];
   const emojis = ["😁", "🙂", "😐", "😤"];
   return `
         <table>
           <tr>
            <th>Campaign</th>
-            ${emojis.map(key => `<th>${key}</th>`).join("")}
+           ${emojis.map(key => `<th>${key}</th>`).join("")}
+           <th>total</th>
           </tr>
           ${voteData
             .map(
               row => `
                 <tr>
                   <td class="table__light">${row.campaignDate}</td>
-                  ${keys.map(key => `<td>${row.counts[key]}</td>`).join("")}
+                  ${keys
+                    .map(key => `<td>${row.counts[key] ?? 0}</td>`)
+                    .join("")}
+                  <td class="table__light">${
+                    row.counts.total
+                      ? row.counts.total
+                      : keys.reduce(
+                          (total, key) => total + (row.counts[key] || 0),
+                          0
+                        )
```

Comment:
Very nice! I like that it falls back on the sum if the database does not have the result already.

One suggestion though: if total is zero, then the sum is computed. This could be avoided like this:
```suggestion
                    typeof row.counts.total === "number"
                      ? row.counts.total
                      : keys.reduce(
                          (total, key) => total + (row.counts[key] || 0),
                          0
                        )
```